src/main/java/controller/AttendanceController.java

```diff
@@ -0,0 +1,145 @@
+package controller;
+
+import domain.AttendanceManager;
+import domain.AttendanceStatistics;
+import domain.Penalty;
+import domain.Records;
+import domain.StatisticsResult;
+import domain.TimeAndStatus;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Map;
+import view.InputView;
+import view.OutputView;
+
+public class AttendanceController {
+
+    private final int DEFAULT_YEAR = 2024;
+    private final int DEFAULT_MONTH = 12;
+    private final int DEFAULT_DAY = 13;
+    private final String INPUT_DATE_FORMAT = "%04d-%02d-%02d";
+    private final String INPUT_TIME_FORMAT = "%02d:%02d";
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final AttendanceManager attendanceManager;
```

Comment:
상수는 static을 붙여줍시다 :)
```suggestion
    private final int DEFAULT_YEAR = 2024;
    private final int DEFAULT_MONTH = 12;
    private final int DEFAULT_DAY = 13;
    private final String INPUT_DATE_FORMAT = "%04d-%02d-%02d";
    private final String INPUT_TIME_FORMAT = "%02d:%02d";
    
    private final InputView inputView;
    private final OutputView outputView;
    private final AttendanceManager attendanceManager;
```

그리고 상수와 일반 변수 사이엔 개행을 넣는 편이에요